### ISSUE TITLE
cydia release?

### ISSUE THREAD

**Issue by Alekemon** [lufinkey/iSSB#19]
idk, just making sure if corneria was enough for a new update

**Comment by lufinkey** [comment]
I'll look into maybe updating it in Cydia, but I have zero control over the distribution anywhere else. It's up to those sites if they want to recompile it with the updates.

**Comment by lufinkey** [comment]
I also don't really want to push an update and give a bunch of people the impression that I'm gonna be updating this regularly. If you want to keep updating it I can push an update, but I don't really want to keep having to do work for this app. I just have a lot of other stuff I'm trying to spend my time on and I don't want to keep having to pull down and test the app, correct changes, build releases, etc.

**Comment by Alekemon** [comment]
Yeah, I'm now working on more characters and more stages, as well as a classic mode (but that's gonna take long).  I can keep up with iSSB perfectly.

**Comment by Alekemon** [comment]
So please, will you post the update? I'll update regularly

**Comment by lufinkey** [comment]
If you add end up adding a decent amount more then I will but right now I just have a lot going on.

If you're really serious about adding more content I can try to put together a PR checklist for when you make changes. I'd really recommend learning how to create a new branch in git so you can manage different changes separately. The thing that will probably speed things up the most when you make a PR is if you're willing to attach video / screenshots showing the character or stage working, showing where the wireframe regions / boundaries are, all the different moves in different ways etc so I don't have to necessarily load the game up myself and test everything a bunch to make sure it works right.

It's really cool that you're interested in contributing and i definitely want to push releases if you're willing to contribute more, I'd just rather push those changes as one or two releases in bulk, that way it can be fully tested and adjusted and all of that, and doing that takes time.

**Comment by Alekemon** [comment]
how much characters and stages are necessary for one update more or less?

**Comment by lufinkey** [comment]
How many characters or stages were you planning on adding? I feel like 3 is probably a good number of characters before I'd feel compelled to update, but are you willing to fully test them, make adjustments to the sprites, discuss movesets beforehand etc? Bc I don't wanna just take Sprite rips and shove them into the game. I'd really want any new characters to feel stable and well playable.

**Comment by Alekemon** [comment]
Sure, I'm planning to add Samus, Jigglypuff and Mega Man. But fully playable with new movesets.

**Comment by lufinkey** [comment]
When you say new movesets, do you mean different than the ones in the smash
games? Because I kinda think for the most part for existing characters we
should stick to the moveset from the games. I'm definitely open to making
it different, but only really if it's for a real reason.

**Comment by lufinkey** [comment]
At the very least for the characters in Brawl or before. After brawl I don't mind as much since the roster gets a little crazy

**Comment by Alekemon** [comment]
I meant movesets that aren't clones of already existing characters in iSSB

**Comment by lufinkey** [comment]
Oh awesome. So they match the smash games?

**Comment by Alekemon** [comment]
Of course

**Comment by lufinkey** [comment]
Nice

**Comment by lufinkey** [comment]
btw, you don't need to re-fork iSSB every time. You can add this repo as a `remote` in the git command line (maybe call it `lufinkey`) and then just pull directly from the remote master. It'll update your branch directly
```
git remote add lufinkey https://github.com/lufinkey/iSSB.git
git pull lufinkey master
```
